### DISCUSSION TITLE
Add Option to Display Frontmatter in Shared Note

### DISCUSSION THREAD

**Discussion by poetex** [alangrainger/share-note#129] [Vote for upcoming features]
The Frontmatter contains a lot of identifying information that I would like to display when sharing a note.

Request:
Add an option to display the frontmatter in the shared note (preferably rendered in the same format).

Current workaround:
Recreate the front matter with `app` and `tp` method calls in a user generated table/list of values. This is clunky and redundant.

**Comment by alangrainger** [comment]
This option already exists. Turn this off in the Settings: 

![Screenshot_20241005-073654](https://github.com/user-attachments/assets/8ce9645d-94a2-403e-ad97-2bb8efe06fcb)

**Reply by poetex** [reply]
I figured this had to exist as a feature, but I wasn't finding it in the settings. Thanks for a quick and helpful response @alangrainger. I'm impressed at your responsiveness throughout the threads for this plugin!

**Reply by alangrainger** [reply]
Would it make more sense if I inverted the option, so that it's:

> Include frontmatter with the published note (default: false)

?
